src/checkstyle/checkstyle-header.txt

```diff
@@ -13,5 +13,4 @@
 ^\Q * See the License for the specific language governing permissions and\E$
 ^\Q * limitations under the License.\E$
 ^\Q */\E$
-^$
```

Comment:
Good work!! - since you've automated the fix, we might as well include this in the rule.